fedimint-core/src/util/backon.rs

```diff
@@ -0,0 +1,44 @@
+use std::time::Duration;
```

Comment:
I guess this is OK, but I fear this module having the exact same name as 3rd party crate is going to be a source of confusion.

Time will tell I guess.

Reply:
Renamed to `backoff_util`